dataland-frontend/src/components/pages/SearchCompaniesForFrameworkData.vue

```diff
@@ -90,15 +118,28 @@ export default defineComponent({
       latestScrollPosition: 0,
       currentSearchBarInput: "",
       currentFilteredFrameworks: [] as Array<DataTypeEnum>,
+      currentFilteredCountryCodes: [] as Array<string>,
+      currentFilteredSectors: [] as Array<string>,
+      currentCombinedFilter: {
+        companyNameFilter: "",
+        frameworkFilter: [],
+        sectorFilter: [],
+        countryCodeFilter: [],
```

Comment:
Nach etwas genauerem Hinsehen frage ich mich gerade, wieso wir überhaupt den combinedFilter unbedingt brauchen.
Wir könnten doch auch in die SearchBar die current[...] Werte über einen bind einzeln reingeben (quasi analog zu dem FilterContainer).
Dann würden wir uns eine Menge Code ersparen bei gleicher Funktion.

Reply:
Dann würden alle Watcher wegfallen und auch die "updateCombinedFilter" Methode würde wegfallen etc.

Reply:
Ich habe herumprobiert und auch Alternativlösungen bruachen erheblich viel Logik, sodass kien Mehrwert ensteht, von der Lösung hier abzuweichen.
Das größte Problem ist das "nur einmal" reagieren auf mehrere Änderungen.
Und eine query über refs nach unten zu schicken ist wohl auch keine good practice, wegen den Lifecycle Problemen.